config.toml

```
@@ -286,3 +286,14 @@ time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
 
+[languages.pl]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
```

nit: This line is still English.

---

Almost all other localizations keep this slogan untranslated. I'm trying to be consistent here.

---

> Almost all other localizations keep this slogan untranslated.

That's true, but it's a bug. The slogan should be localized.

---

I've sent in a bunch of pull requests to fix the other localizations.

---

As discussed on slack #kubernetes-docs-pl:
```suggestion
description = "Produkcyjny system zarządzania kontenerami"
```